app/commands/flows.go

```diff
@@ -2,11 +2,13 @@ package commands
 
 import (
 	"fmt"
-	"github.com/codegangsta/cli"
-	"github.com/codelingo/lingo/app/util"
-	"github.com/juju/errors"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/codelingo/lingo/app/util"
+
```

Comment:
Hmmm, shouldn't goimports put all the github.com imports in one block?

Reply:
Convention is to split project imports from 3rd party imports.